### ISSUE TITLE
Assembly 'Assets/Firebase/Editor/Firebase.Editor.dll' will not be loaded due to errors: Unable to resolve reference 'UnityEditor.iOS.Extensions.Xcode'. Is the assembly missing or

### ISSUE THREAD

**Issue by Sacristan** [firebase/firebase-unity-sdk#218]
Please allow us to support only one platform. For example, Meta Quest usecase where Unity projects needs to have just Android support

Therefore, Editor scripts should check whether build module exists (Android, iOS)

**Comment by DanForstinger** [comment]
We are seeing this issue as well. We run CI builds for android and IOS, but only have references to a single platform at a time and no way to import the ios module when building android. Any word on a fix?

**Comment by wanglailai** [comment]
Same issue here.   Some people just run project  in Editor, and they do not want to install iOS or Android Module.

The solution is  modify editor/crashlytics/src/iOSPostBuild.cs file and build the sdk yourself.
```
-    private static void SetupGUIDForSymbolUploads(UnityEditor.iOS.Xcode.PBXProject pbxProject,
+    private static void SetupGUIDForSymbolUploads(object projectObj,
                                                   string completeRunScriptBody,
 string targetGuid) {
+      var pbxProject = (UnityEditor.iOS.Xcode.PBXProject)projectObj;
       try {
....
-    private static string GetUnityFrameworkTargetGuid(UnityEditor.iOS.Xcode.PBX
Project project) {
-      // var project = (UnityEditor.iOS.Xcode.PBXProject)projectObj;
+    private static string GetUnityFrameworkTargetGuid(object projectObj) {
+      var project = (UnityEditor.iOS.Xcode.PBXProject)projectObj;

```

**Comment by chkuang-g** [comment]
Ah, thanks for reporting this issue.  

This is definitely an oversight from our side.  And thanks for the suggested workaround.

Let me bring Crashlytics team in to resolve this issue.

@samedson Could you help to update the Crashlytics Editor tool?

**Comment by chkuang-g** [comment]
This should be included in the next release `9.5.0`.  If the issue persists, I recommend to open a new issue and reference this one for better visibility.
